crates/bevy-inspector-egui/Cargo.toml

```diff
@@ -21,25 +21,28 @@ highlight_changes = []
 
 [dependencies]
 bevy-inspector-egui-derive = { version = "0.24.0", path = "../bevy-inspector-egui-derive" }
-bevy_app = { version = "0.13" }
-bevy_asset = { version = "0.13" }
-bevy_core = { version = "0.13" }
-bevy_ecs = { version = "0.13" }
-bevy_hierarchy = { version = "0.13" }
-bevy_log = { version = "0.13" }
-bevy_math = { version = "0.13" }
-bevy_reflect = { version = "0.13" }
-bevy_time = { version = "0.13" }
-bevy_utils = { version = "0.13" }
-bevy_window = { version = "0.13" }
-
-bevy_core_pipeline = { version = "0.13" }
-bevy_pbr = { version = "0.13", optional = true }
-bevy_render = { version = "0.13" }
+bevy_app = { version = "0.14.0-rc.2" }
+bevy_asset = { version = "0.14.0-rc.2" }
+bevy_color = { version = "0.14.0-rc.2" }
+bevy_core = { version = "0.14.0-rc.2" }
+bevy_ecs = { version = "0.14.0-rc.2" }
+bevy_hierarchy = { version = "0.14.0-rc.2" }
+bevy_log = { version = "0.14.0-rc.2" }
+bevy_math = { version = "0.14.0-rc.2" }
+bevy_reflect = { version = "0.14.0-rc.2" }
+bevy_state = { version = "0.14.0-rc.2" }
+bevy_time = { version = "0.14.0-rc.2" }
+bevy_utils = { version = "0.14.0-rc.2" }
+bevy_window = { version = "0.14.0-rc.2" }
+
+bevy_core_pipeline = { version = "0.14.0-rc.2" }
+bevy_pbr = { version = "0.14.0-rc.2", optional = true }
+bevy_render = { version = "0.14.0-rc.2" }
```

Comment:
lines like this make me want a *constant definition* feature for `cargo.toml` :')

Reply:
Oh please god no, it was horrible enough with maven, that'd mean 0 lsp capabilities, statical analysis would be a lot harder and you can no longer just look at them to figure out whats what.

You can always just search and replace if you want to bulk edit.

Reply:
fair enough 👍